stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -117,6 +117,7 @@
     private SourceReceiver mReceiver;
     private SourceRedirect mRedirect;
     private @SourceStatus String mStatus;
+    private Map<String, Object> mSourceTypeData;
```

Comment:
just want to note that we create first-class models for this hash on the iOS SDK for `card` and `sepa_debit` sources [0] – totally cool if you're planning on doing this in a follow-up PR, just wanted to remind

[0] https://github.com/stripe/stripe-ios/blob/master/Stripe/PublicHeaders/STPSource.h#L167-L177